indico/web/templates/forms/radio_buttons_widget.html

```diff
@@ -3,12 +3,33 @@
 {% set vertical_alignment = field.option_orientation == 'vertical' %}
 
 {% block html %}
-    <div class="{{ 'i-group' if vertical_alignment else 'group i-selection' }}">
+    <div id="{{ field.id }}" class="{{ 'i-group' if vertical_alignment else 'group i-selection' }}">
         {% for subfield in field %}
             <div class="{{ 'i-radio' if vertical_alignment else 'inline-vcentered'}} radio-item-{{ loop.index0 }}">
                 {{ subfield(**input_args) }}
                 {{ subfield.label(class_=('i-button' if not vertical_alignment)) }}
             </div>
         {% endfor %}
     </div>
+    <script>
+        (function() {
+            'use strict';
+
+            $('#{{ field.id }} input').bind('invalid', function(evt) {
+                $('#{{ field.id }} div:last-child').qtip({
+                    content: {
+                        text: $T.gettext(evt.target.validationMessage)
```

Comment:
using i18n here is kind of pointless since the messages won't be in the dictionary